tuna/miopen/utils/parsing.py

```diff
@@ -107,7 +110,7 @@ def parse_2d(key, group_count):  #pylint: disable=too-many-locals
   fil_h, fil_w = tmp[3].split('x')
   batchsize = tmp[7]
   pad_h, pad_w = tmp[8].split('x')
-  conv_stride_w, conv_stride_h = tmp[9].split('x')
+  conv_stride_h, conv_stride_w = tmp[9].split('x')
   dilation_h, dilation_w = tmp[10].split('x')
```

Comment:
So things are no longer backward compatible now?

Reply:
This parsing script was used historically to import configs. And you can see it reference in the driver class still. This does mean that any configs imported from a fdb would be inaccurate, they would be different from the expected config if h != w.  However the configs in the conv_config table are still valid (if unexpected) and the results produced by them are valid and have valid fdb keys. Configs imported by driver command are unaffected by this.
This was discovered when updating the export pdb functionality. The configs produced using this parsing method did not match those observed by the previous method.

Reply:
I see